text_machina/src/exploration/mixcase.py

```diff
@@ -0,0 +1,56 @@
+from typing import Any, Dict, List
+
+from datasets import Dataset
+from rich.markup import escape
+from rich.panel import Panel
+
+from .base import Explorer
+
+
+class MixCaseExplorer(Explorer):
```

Comment:
Since this includes many spans with different annotations, consider including a panel that adds that information (e.g. some small thing with human + the model name highlighted with their respective colors), since otherwise its difficult to know which spans are human vs generated. This is different to the boundary task since there we know the human spans are always the prefixes.

Reply:
done. Changed the outer title to "Mixcase text in domain ...", and added to the inner one the distinction with colors.